examples/hitl/rearrange_v2/ui.py

```diff
@@ -696,20 +705,29 @@ def _update_selected_object_ui(self):
                 # Get the contextual information.
                 color_ui_valid = [0.2, 1.0, 0.2, 1.0]
                 color_ui_invalid = [1.0, 0.2, 0.2, 1.0]
+                color_ui_info = None
+                color_ui_object_info = [0.8, 0.8, 0.2, 1.0]
```

Comment:
hm maybe we should only have ` color_ui_info` and use it in all places? rather than have ` color_ui_info` and `color_ui_object_info`?

Reply:
Also let;s make these the same as the colors defined at the top of this file, rather than re-define them here?

Reply:
>Also let;s make these the same as the colors defined at the top of this file, rather than re-define them here?

I'll refactor that in a later pass.
1. Those 2 colors are in different formats. This will be unified.
2. The contextual information duplicates the real checks. This will be unified.

Reply:
>hm maybe we should only have  color_ui_info and use it in all places? rather than have  color_ui_info and color_ui_object_info?

I think that it's preferable to have contextual information related to pickable objects yellow, like the pickable object highlights.